### PR TITLE
Adding VerifiedIdentity flow to quick setup modal

### PR DESCRIPTION
**Pull request by Sarahb-da**
Screen Shot 2021-04-20 at 5 07 52 PM

If the user already has a `VerifiedIdentity` `legalName` and `location` will autofill with the contract data. 

Screen Shot 2021-04-20 at 5 10 12 PM

### CODE REVIEW

File: ui2/src/pages/login/QuickSetup.tsx [hunk 1]
```diff
@@ -56,14 +64,19 @@ enum ServiceKind {
   SETTLEMENT = 'Settlement',
 }
 
-interface IQuickSetupData {
+interface IVerifiedIdentityData {
+  legalName?: string;
+  location?: string;
+}
+
+interface IQuickSetupData extends IVerifiedIdentityData {
```
Comment (alexmatson-da) [inline ui2/src/pages/login/QuickSetup.tsx:72]: nice!

File: ui2/src/pages/login/QuickSetup.tsx [hunk 2]
```diff
@@ -80,26 +93,36 @@ const QuickSetupTable = (props: {
...
   const [status, setStatus] = useState<string>();
   const [loading, setLoading] = useState<boolean>(false);
-  const [marketSetupDataMap, setMarketSetupDataMap] = useState<Map<string, string[]>>(new Map());
+  const [marketSetupDataMap, setMarketSetupDataMap] = useState<Map<string, IQuickSetupData>>(
+    new Map()
+  );
+  const [verifiedIdentity, setVerifiedIdentity] = useState<{
+    legalName?: string;
+    location?: string;
+  }>();
```
Comment (alexmatson-da) [inline ui2/src/pages/login/QuickSetup.tsx:102]: `useState<IVerifiedIdentityData>` perhaps?

File: ui2/src/pages/login/QuickSetup.tsx [hunk 3]
```diff
@@ -80,26 +93,36 @@ const QuickSetupTable = (props: {
...
   const custodianOffers = useStreamQueries(CustodianOffer);
   const distributorOffers = useStreamQueries(DistributorOffer);
   const settlementOffers = useStreamQueries(SettlementOffer);
   const exhangeOffers = useStreamQueries(ExchangeOffer);
   const matchingOffers = useStreamQueries(MatchingOffer);
 
-  const selectedParty = quickSetupData?.party;
-  const selectedServices = quickSetupData?.services || [];
+  const verifiedIdentities = useStreamQueries(VerifiedIdentity);
+  const verifiedIdeneityRequests = useStreamQueries(IdentityVerificationRequest);
```
Comment (alexmatson-da) [inline ui2/src/pages/login/QuickSetup.tsx:125]: typo 

```suggestion
  const verifiedIdentityRequests = useStreamQueries(IdentityVerificationRequest);
```

File: ui2/src/pages/login/QuickSetup.tsx [hunk 4]
```diff
@@ -285,14 +404,17 @@ const QuickSetupTable = (props: {
...
+      provider: operator,
+      customer: provider,
+    });
 
     Promise.all(
-      selectedServices.map(async service => {
+      services.map(async service => {
         if (findExistingOffer(service)) {
           return;
         }
```
Comment (alexmatson-da) [inline ui2/src/pages/login/QuickSetup.tsx:420]: Should the Regulator service offer be a part of the `findExistingOffer()` and switch case below?

**Review by alexmatson-da — APPROVED** [review]
This is great, thanks for taking this on!

File: ui2/src/pages/login/QuickSetup.tsx (hunk 2 above)
Comment (Sarahb-da) [reply ui2/src/pages/login/QuickSetup.tsx:102]: oh woops, thanks

File: ui2/src/pages/login/QuickSetup.tsx (hunk 4 above)
Comment (Sarahb-da) [reply ui2/src/pages/login/QuickSetup.tsx:420]: the regulator service wont be in the list of services, so it doesn't need to be in the switch case - I could check for existing offers though. Will add that in

**Review by alexgraham-da — APPROVED** [review]
looks good!

**Pull request merged**